### ISSUE TITLE
Add uninstaller for Solana tooling

### ISSUE THREAD

**Issue by AurelienFT** [solana-labs/solana#19744]
#### Problem

I want to uninstall Solana from my workstation. Is that possible ?

#### Proposed Solution

Document on how to do it.

**Comment by CapacitorDev** [comment]
hi @AurelienFT , did you find a way to remove it?

**Comment by AurelienFT** [comment]
> hi @AurelienFT , did you find a way to remove it?

No :/

**Comment by r3ndd** [comment]
Issues like those in #21053 and #23294 would be solved by adding a comprehensive uninstaller that clears the cache. It might be a good idea to add one to the CLI.

**Comment by Clariti23** [comment]
I too would appreciate an uninstaller in the CLI.

**Comment by 1scriptz** [comment]
This would be a greatly appreciated feature/bit of documentation - being I somehow installed v1.15 of the CLI.

Zero clue how to uninstall it.

**Comment by eulersexception** [comment]
I tried to follow the installation guide and was confronted with this [problem](https://github.com/anza-xyz/platform-tools/issues/87). To clean up my system, I had to dig deeper and found this issue. 

Assuming the output of `which solana` contains the basepath `$HOME/.local/share/solana` 
I executed this commands: 
```
rustup toolchain uninstall solana 
rm -rf ~/.local/share/solana
rm -rf ~/.config/solana 
```

To get more info: `rustup toolchain help` 

System: macOS Ventura 13.7.1
Disclaimer: no Rust expert.
